docs/distributions/fedora/installation.md

```diff
@@ -0,0 +1,47 @@
+# Download the latest safe release
+
+Thanks so Mike for building. Download here. [https://github.com/mikeeq/mbp-fedora](https://github.com/mikeeq/mbp-fedora)
```

Comment:
Maybe change this to "You can download a live iso here (link)"

Reply:
done